### PR TITLE
clarify that there are two different auto-merge approaches

### PR DESCRIPTION
**Pull request by burnettk**
one uses github actions and the other does not. this change, had it been in place, would have saved me some time.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@burnettk 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md [hunk 1]
```diff
@@ -452,9 +452,9 @@ jobs:
 
 ### Enable auto-merge on a pull request
 
-If you want to auto-merge your pull requests, you can use {% data variables.product.prodname_dotcom %}'s auto-merge functionality. This enables the pull request to be merged when all required tests and approvals are successfully met. For more information on auto-merge, see "[Automatically merging a pull request](/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)."
+If you want to allow maintainers to mark certain pull requests for auto-merge, you can use {% data variables.product.prodname_dotcom %}'s auto-merge functionality. This enables the pull request to be merged when all required tests and approvals are successfully met. For more information on auto-merge, see "[Automatically merging a pull request](/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)."
 
-Here is an example of enabling auto-merge for all patch updates to `my-dependency`:
+You can instead use github actions and the github CLI. Here is an example that auto merges all patch updates to `my-dependency`:
```
Comment (mchammer01) [inline content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md:457]:
```suggestion
You can instead use {% data variables.product.prodname_actions %} and the {% data variables.product.prodname_dotcom %} CLI. Here is an example that auto merges all patch updates to `my-dependency`:
```

**Review by mchammer01 — COMMENTED** [review]
@burnettk 👋🏻 - thanks for your contribution ✨ 
I've made an editorial suggestion to use variables. I'll apply it and then will approve this work 😃

File: content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md [hunk 2]
```diff
@@ -452,9 +452,9 @@ jobs:
...
+If you want to allow maintainers to mark certain pull requests for auto-merge, you can use {% data variables.product.prodname_dotcom %}'s auto-merge functionality. This enables the pull request to be merged when all required tests and approvals are successfully met. For more information on auto-merge, see "[Automatically merging a pull request](/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)."
 
-Here is an example of enabling auto-merge for all patch updates to `my-dependency`:
+You can instead use {% data variables.product.prodname_actions %} and the {% data variables.product.prodname_dotcom %} CLI. Here is an example that auto merges all patch updates to `my-dependency`:
```
Comment (mchammer01) [inline content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md:457]:
```suggestion
You can instead use {% data variables.product.prodname_actions %} and the {% data variables.product.prodname_cli %}. Here is an example that auto merges all patch updates to `my-dependency`:
```

**Review by mchammer01 — APPROVED** [review]
LGTM 💖

**Comment by mchammer01** [comment]
The staging preview doesn't seem to be working 😞 
I'm going to merge this.

**Pull request merged**